### DISCUSSION TITLE
How do I update Lat & Lon via API?

### DISCUSSION THREAD

**Discussion by tilusnet** [photoprism/photoprism#4434] [Q&A] [answered]
I'd like to do bulk Latitude and Longitude updates on a set of images via API.
I found an `/api/v1/geo` endpoint but all it does it's searching.

Any tips please?

Thanks.

**Comment by graciousgrey** [comment, marked as answer]
Currenlty you can change the Lat/Lng of single photos using the /api/v1/photos endpoint.

https://github.com/photoprism/photoprism/blob/develop/internal/api/photos.go

Batch editing endpoints will be added when we implement Batch Editing which is planned for later this year.

**Reply by lastzero** [reply]
@tilusnet If using the REST API is not critical for you, you can also connect directly to the underlying SQL database to batch update the GPS position in the `photos` table.
